src/acceptance-tests-brain/test-scripts/010_insecure_registry_test.rb

```diff
@@ -69,31 +68,17 @@
   at_exit do
     set errexit: false do
       puts "........................................................................... SHUTDOWN"
-      run "cf delete -f secure-registry"
-      run "cf delete -f insecure-registry"
-      run "cf delete -f uploader"
+      %w(secure-registry insecure-registry uploader).each do |app|
```

Comment:
We currently do not have a `secure-registry` in the set.

Reply:
Ok. Accepting as is, because this code run on `atexit`, thus the failure for `secure` does not matter.